lib/install/stylesheets/components.scss

```diff
@@ -0,0 +1,4 @@
+@import "tailwindcss/components";
+
+// Import components added to `./components/` here
+// https://tailwindcss.com/docs/extracting-components
```

Comment:
@dixpac why do we need the special components folder? There is also no folder to structure the base styles (and I would do this too).

I see two ways here:

1. Everybody should add the own structure
2. We give them a structure (CoC). Just to do that, I am not too much experienced with tailwind ;)